org.ampr.ct1enq.tags.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id" : "org.ampr.ct1enq.tags",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "tags",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--filesystem=home"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "tags",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/phastmike/tags.git"
```

Comment:
tag+commit is needed.

Reply:
Add developer_name in appdata

Pull the screenshot from a tag instead of main https://github.com/phastmike/tags/blob/b4fb345c2ca843976c895960a7f3d51b68567175/data/org.ampr.ct1enq.tags.appdata.xml.in#L27C87-L27C87

Reply:
Roger that, i'll change it